WordPress/Classes/ViewRelated/Reader/ReaderPostCardCell.swift

```diff
@@ -139,6 +148,8 @@ private struct Constants {
         setupSummaryLabel()
         setupAttributionView()
         adjustInsetsForTextDirection()
+
+        topicsCollectionView.topicDelegate = self
```

Comment:
Maybe move that to `configureTopicsCollectionView`?

Reply:
👍